chassis/coordinator/src/main/java/com/griddynamics/jagger/coordinator/DefaultNodeContext.java

```diff
@@ -46,11 +46,6 @@ public NodeId getId() {
         if (services.containsKey(clazz)) {
             return (T) services.get(clazz);
         }
-        try {
-            return clazz.newInstance();
-        } catch (Exception e) {
-            log.warn("Didn't manage to create an instance of {}", clazz, e);
-        }
```

Comment:
it is not needed any more. now providers are used to create instances not reflection